logback-core/src/test/java/ch/qos/logback/core/rolling/helper/RollingCalendarTest.java

```diff
@@ -30,7 +31,8 @@
 
 public class RollingCalendarTest {
 
-  String dailyPattern = "yyyy-MM-dd";
+   String dailyPattern = "yyyy-MM-dd";
+   private static final String ISO_FORMAT = "yyyy-MM-dd'T'HH:mm:ss.SSS zzz";
```

Comment:
this  isn't ISO format. It has to be XXX.

Reply:
Changed it just to "DATE_FORMAT" to avoid any confusions. It's only used in test to go from string to date and back